### PR TITLE
[12.0] [ADD] Send credit control emails in job batches

### PR DESCRIPTION
**Pull request by qgroulard**

### CODE REVIEW

File: account_credit_control_queue_job/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+{
+    "name": "Account Credit Control Queue Job",
+    "summary": """
+        This addon adds the opportunity to run
+        some credit control tasks in jobs""",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://acsone.eu",
```
Comment (nikul-serpentcs) [inline account_credit_control_queue_job/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/credit-control",
```

**Review by sbidoul — APPROVED** [review]
LGTM (code review)

**Comment by sbidoul** [comment]
@gva-acsone you have tested this, right?

**Review by gva-acsone — APPROVED** [review]
@sbidoul, yes! LGTM functional test

**Pull request merged**